### DISCUSSION TITLE
featurerequest: customize menu

### DISCUSSION THREAD

**Discussion by 13minutes-yt** [doublecmd/doublecmd#1320] [Suggestions]
One of the key features of TC is the ability to customize the menu through the language text file. Is it difficult for DC to implement that strategy which will make Double commander more customizable.

**Comment by JonnyTech** [comment]
Before I became a Double Commander convert, I also used Total Commander. The "Extended Menus" were always a joy. See https://ghisler.ch/board/viewtopic.php?t=33740 for some examples. As a (mostly) keyboard user, the customisation was very convenient.

**Comment by wilfz** [comment]
I also requested this feature (so this discussion is a duplicate of https://github.com/doublecmd/doublecmd/discussions/1141 and vice versa). Actually it's one of the reasons I stick with TC on windows. But on Linux I do use doublecmd and the possibility of configuring the menu is the one thing I miss most. And, no, configurable button bars are not an adequate substitute  .

**Reply by SAKryukov** [reply]
I've just supported your idea from another point of view: the possibility to ***unify*** TC and Double Commander main menus as much as possible. It requires menu customization.
Thank you.
—SA

**Comment by SAKryukov** [comment]
I would like to add my two cents:

There is another reason to customize the main menu. Say, I keep using TC on my Windows systems and Double Commander on Linux. I don't like the default TC main menu, because some important commands are hidden. Even though they are visible for Double Commander, I would prefer to ***unify*** TC and Double Commander main menus as much as possible. To achieve that, I would need menu customization for Double Commander.

I think many multi-platform users will agree with my idea.
